GPSTest/src/main/res/values-cs-rCS/strings.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+<string name="app_feedback_email">barbeaudev@gmail.com</string>
```

Comment:
None of the strings in `do_not_translate.xml` should be in this `strings.xml` file - could you please remove them?